origin-js/src/ipfsInterface/adapters/listing/v1-listing-adapter.js

```diff
@@ -81,8 +82,8 @@ export default class ListingAdapterV1 extends AdapterBase {
         ? new Money(ipfsData.commission)
         : null
       listing.commissionPerUnit = ipfsData.commissionPerUnit
-      ? new Money(ipfsData.commissionPerUnit)
-      : null
+        ? new Money(ipfsData.commissionPerUnit)
+        : null
```

Comment:
I'm a little confused about the use of `new Money()` as it is used in this file (not just from this new use of it). 

To me it looks like you'd at least have to pass it an object with a key called `amount` in order for it to return anything useful, and ideally, you'd also pass it another key called `currency`, which differs between `OGN` and `ETH` in this file (price is normally ETH and commission is normally OGN), but that's not defined in this file...

How does `new Money()` return a useful object without being explicitly passed `amount` or `currency`?

Reply:
commissionPerUnit is an object with a amount and currency. See [listing schema](https://github.com/OriginProtocol/origin/blob/master/origin-js/src/ipfsInterface/schemas/listing.json#L88).

Reply:
Ahh, of course - thanks for explaining